src/test/java/de/fearnixx/jeak/test/plugin/Mp3PlayerPlugin.java

```diff
@@ -0,0 +1,154 @@
+package de.fearnixx.jeak.test.plugin;
+
+import de.fearnixx.jeak.IBot;
+import de.fearnixx.jeak.event.bot.IBotStateEvent;
+import de.fearnixx.jeak.reflect.Inject;
+import de.fearnixx.jeak.reflect.JeakBotPlugin;
+import de.fearnixx.jeak.reflect.Listener;
+import de.fearnixx.jeak.service.command.CommandException;
+import de.fearnixx.jeak.service.command.ICommandService;
+import de.fearnixx.jeak.teamspeak.cache.IDataCache;
+import de.fearnixx.jeak.test.AbstractTestPlugin;
+import de.fearnixx.jeak.voice.connection.IVoiceConnection;
+import de.fearnixx.jeak.voice.connection.IVoiceConnectionService;
+import de.fearnixx.jeak.voice.event.IVoiceConnectionTextMessageEvent;
+import de.fearnixx.jeak.voice.sound.IMp3AudioPlayer;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.apache.commons.lang3.tuple.Pair;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.*;
+import java.util.concurrent.TimeoutException;
+
+@JeakBotPlugin(id = "mpThreePlayer")
+public class Mp3PlayerPlugin extends AbstractTestPlugin {
```

Comment:
I really love how basically simple the actual use looks for the plugin developer :smiley: :+1:   
This should be our main goal in order to compete with other frameworks that have voice functionality.

Reply:
:blue_heart: